main.py

```diff
@@ -52,108 +78,205 @@ def open_browser():
 
 
 def purchase_steps(browser):
+
+    try:
+        # Search for license agreement popup and click it
+        logger.debug("find and accept license agreement")
+        WebDriverWait(browser, TIMEOUT).until(
+            EC.visibility_of_element_located((
+                By.ID,
+                "agree"
+            ))
+        ).click()
+
+        # Search and click Accept button
+        WebDriverWait(browser, TIMEOUT).until(
+            EC.visibility_of_element_located((
+                By.CSS_SELECTOR,
+                ".css-1b2j8lx"
```

Comment:
This won't work, the css names are not permanent. I believe these come from using tools like SASS and is not reliable 
I tested with the XPATH `//button[descendant::span[text()='Accept']]` and it worked fine